datalad/tests/test_tests_utils.py

```diff
@@ -186,11 +161,6 @@ def test_get_resolved_values():
     with patch_config({'datalad.tests.nonetwork': '1'}):
         eq_(_get_resolved_flavors(flavors), ['local'])
 
-        # and one more to see the exception being raised if nothing to teston
-        @with_testrepos(flavors=['network'])
-        def magical():
-            raise AssertionError("Must not be ran")
-        assert_raises(SkipTest, magical)
```

Comment:
@yarikoptic would have appreciated to have this tested as long as we have debian packages built for extensions which still use `@with_testrepos`

Reply:
From my POV the only extension using it is `deprecated`, which would also be the place to migrate this functionality to. @yarikoptic are you aware of other extensions using it?